### PR TITLE
[DROOLS-6219] Centralized common generating code

### PR DESCRIPTION
**Pull request by gitgabrio**
@danielezonca @kostola  @jiripetrlik

This PR centralize generation of code related to pre/post processing in one single class `KiePMMLModelCodegenUtils`.

It requires https://github.com/gitgabrio/drools/pull/3 to be merged before.

### CODE REVIEW

**Review by kostola — APPROVED** [review]
LGTM

File: kie-pmml-trusty/kie-pmml-compiler/kie-pmml-compiler-commons/src/main/java/org/kie/pmml/compiler/commons/builders/KiePMMLModelConstructorBuilder.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+import static org.kie.pmml.commons.Constants.MISSING_DEFAULT_CONSTRUCTOR;
+import static org.kie.pmml.commons.utils.KiePMMLModelUtils.getSanitizedClassName;
+import static org.kie.pmml.compiler.commons.factories.KiePMMLOutputFieldFactory.getOutputFields;
+import static org.kie.pmml.compiler.commons.utils.KiePMMLModelFactoryUtils.addKiePMMLOutputFieldsPopulation;
+import static org.kie.pmml.compiler.commons.utils.KiePMMLModelFactoryUtils.addTransformationsInClassOrInterfaceDeclaration;
+import static org.kie.pmml.compiler.commons.utils.KiePMMLModelFactoryUtils.setKiePMMLModelConstructor;
+import static org.kie.pmml.compiler.commons.utils.ModelUtils.getTargetFieldName;
+
+public class KiePMMLModelConstructorBuilder {
```
Comment (danielezonca) [inline kie-pmml-trusty/kie-pmml-compiler/kie-pmml-compiler-commons/src/main/java/org/kie/pmml/compiler/commons/builders/KiePMMLModelConstructorBuilder.java]: What about make this class a `KiePMMLModelCodegenUtils` or similar with a static method `initialize(...)`?

**Review by jiripetrlik — APPROVED** [review]
LGTM

File: kie-pmml-trusty/kie-pmml-compiler/kie-pmml-compiler-commons/src/main/java/org/kie/pmml/compiler/commons/builders/KiePMMLModelConstructorBuilder.java (hunk 1 above)
Comment (gitgabrio) [reply kie-pmml-trusty/kie-pmml-compiler/kie-pmml-compiler-commons/src/main/java/org/kie/pmml/compiler/commons/builders/KiePMMLModelConstructorBuilder.java]: @danielezonca 
done

**Review by danielezonca — APPROVED** [review]

**Pull request merged**